lib/flame/dispatcher.rb

```diff
@@ -86,6 +86,12 @@ def config
 			@app_class.config
 		end
 
+		## Available routes endpoint
+		def available_endpoint
+			@available_endpoint ||=
+				@app_class.router.routes.endpoint(*request.path.parts)
+		end
```

Comment:
You will try to find endpoint in each method calling if it not found :(

Reply:
Changed.